scripts/release_issue_status/main.py

```diff
@@ -124,9 +141,19 @@ def main():
             item.bot_advice = 'new issue and better to confirm quickly.'
         elif item.delay_from_latest_update >= 7:
             item.bot_advice = 'delay for a long time and better to handle now.'
-        elif item.delay_from_create_date >= 30:
-            item.bot_advice = 'delay for a month and better to close.'
-
+            
+            
+        if item.delay_from_create_date >= 30 and item.language == 'Python' and not item.whether_author_comment and '30days attention' not in item.labels:
+            item.labels.append('30days attention')
+            item.issue_object.set_labels(*item.labels)
+            issue.issue_object.edit(state='close')
+        elif item.delay_from_create_date >= 15 and item.language == 'Python' and not item.whether_author_comment and '15days attention' not in item.labels:
+            item.issue_object.create_comment('hi @{} ,this release-request has been delayed more than 15 days,'
+                                             ' please deal with it ASAP'.format(item.author))
+            item.labels.append('15days attention')
```

Comment:
Add more hint : `we will close the issue if there is still no response after 15 days!`